rest/resource-server/src/docs/asciidoc/releases.adoc

```diff
@@ -239,7 +239,9 @@ have been successful.
 ==== Trigger FOSSology process
 
 A `GET` request will allow you to Trigger FOSSology process for a release. +
-Please set the request parameter `&markFossologyProcessOutdated=<true|false>` to set previous FOSSology process outdated and generate new.
+Please set the request parameter  +
+   `&markFossologyProcessOutdated=<true|false>` to set previous FOSSology process outdated and generate new. +
+   `&uploadDescription=uploadDescription` with the upload description to FOSSology +
```

Comment:
Please remove the `+` at the end. I think it's not necessary.

Reply:
`+` is added to add newline/line break. Without it comes in a single line
Shall I remove it?